contracts/test/harvest/simple-harvester.mainnet.fork-test.js

```diff
@@ -30,29 +31,138 @@ describe("ForkTest: SimpleHarvester", function () {
     );
   });
 
-  it("Should support Strategy as governor", async () => {
+  // --- Harvest and Transfer Rewards ---
+  it("Should Harvest and transfer rewards (out of WETH) as strategist", async () => {
+    const { simpleOETHHarvester, convexEthMetaStrategy, crv, strategist } =
+      fixture;
+
+    await ensureStrategyIsSupported(
+      simpleOETHHarvester,
+      convexEthMetaStrategy.address,
+      strategist
+    );
+
+    const balanceBeforeCRV = await crv.balanceOf(strategist.address);
+    // prettier-ignore
+    await simpleOETHHarvester
+      .connect(strategist)["harvestAndTransfer(address)"](convexEthMetaStrategy.address);
+
+    const balanceAfterCRV = await crv.balanceOf(strategist.address);
+    expect(balanceAfterCRV).to.be.gt(balanceBeforeCRV);
+  });
+
+  it("Should Harvest and transfer rewards (only of WETH) as strategist", async () => {
+    const {
+      simpleOETHHarvester,
+      nativeStakingSSVStrategy,
+      weth,
+      strategist,
+      oethFixedRateDripper,
+      josh,
+      nativeStakingFeeAccumulator,
+    } = fixture;
+
+    // Send ETH to the FeeAccumulator to simulate yield.
+    await josh.sendTransaction({
+      to: nativeStakingFeeAccumulator.address,
+      value: oethUnits("1"),
+    });
+
+    const balanceBeforeWETH = await weth.balanceOf(
+      oethFixedRateDripper.address
+    );
+    // prettier-ignore
+    await simpleOETHHarvester
+      .connect(strategist)["harvestAndTransfer(address)"](nativeStakingSSVStrategy.address);
+
+    const balanceAfterWETH = await weth.balanceOf(oethFixedRateDripper.address);
+    expect(balanceAfterWETH).to.be.gt(balanceBeforeWETH);
```

Comment:
If 1 ETH is sent to the fee accumulator the balanceAfterWETH should be at least: 
`to.be.gte(balanceBeforeWETH.add(oethUnits("1")))`

Reply:
Indeed! Changed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/bededf793d68b01bc38c55a90d7a7034e46f46bb